install/gcp/gcp_installer.py

```diff
@@ -69,6 +70,15 @@ def run():
                         choices=['client', 'server'],
                         help='Type of the installation, '
                              'either client or server')
+    parser.add_argument('--composite-root-resources',
+                        help='The resource ids to be inventoried.\n'
+                             'Without this flag, the entire org '
+                             'will be attempted.\n'
+                             'Resources must be comma-separated and '
+                             'in the form type/id,\nwhere type is '
+                        'one of organizations, folders, or projects.')
+    parser.add_argument('--project-id',
```

Comment:
should this be --project to be consistent with gcloud binary?

Reply:
TODO coming per above review & comment.

Reply:
I don't see a TODO and not completely sure why we need it for this? I was just asking for a flag rename here /s/project-id/project/